### DISCUSSION TITLE
Artifacts not downloading, where to report?

### DISCUSSION THREAD

**Discussion by ssallmen-pro** [community/community#37208] [Actions] [answered]
I was debugging one of my Actions workflows and it seems that currently it is not possible to download any artifacts from the workflow run page. Does anyone know where should I report this kind of incidents?
It is not concerning only my own repo, but I tried few different random repos and actions runs, and it seems to be a general issue for any repo/workflow. It just keeps on trying to download (blue bar on the top of the page proceeds) but nothing happens.

**Comment by DSDmark** [comment, marked as answer]
Hey @ssallmen-pro,

If you want to **bug report** then visit there:-
[**Bug-Report**](https://support.github.com/contact/bug-report)

---

Or if you want some things else then visit there:-
[**Github-support**](https://support.github.com/)

**Comment by ssallmen-pro** [comment]
Thanks! The issue seems to be fixed now somehow. I just wonder where should I have found that link to GitHub bug report form? I tried to find it from github.com pages and by google, but without any success. But, anyhow, next time I know where to report bugs on GitHub itself.
